scripts/_helpers.py

```diff
@@ -10,12 +10,47 @@
 
 import country_converter as coco
 import geopandas as gpd
-import numpy as np
 import pandas as pd
+import yaml
 
 REGION_COLS = ["geometry", "name", "x", "y", "country"]
 
 
+# world_iso = osm_config["world_iso"]
+# continent_regions = osm_config["continent_regions"]
+# iso_to_geofk_dict = osm_config["iso_to_geofk_dict"]
+# osm_clean_columns = osm_config["osm_clean_columns"]
+
+
+def read_osm_config(*args):
+    """
+    Reads the osm_config.yaml file and returns values based on the provided key
+    arguments.
```

Comment:
What do you think here?
We could add a return statement like:
``` python
if len(args) == 0:
   return osm_config
elif len(args) 1:
   return osm_config[args[0]]
else:
   tuple([osm_config[a] for a in args])
```

This may enable dropping the `[0]` needed when only one argument is passed. What do you think?

Reply:
done.